cmd/opm/alpha/veneer/render.go

```diff
@@ -0,0 +1,82 @@
+package veneer
+
+import (
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	"github.com/operator-framework/operator-registry/alpha/declcfg"
+	"github.com/operator-framework/operator-registry/alpha/veneer/basic"
+	"github.com/operator-framework/operator-registry/cmd/opm/internal/util"
+	containerd "github.com/operator-framework/operator-registry/pkg/image/containerdregistry"
+)
+
+func newBasicVeneerRenderCmd() *cobra.Command {
+	var (
+		veneer basic.BasicVeneer
+		output string
+	)
+	cmd := &cobra.Command{
+		Use:   "render veneer-file...",
+		Short: "Generate a declarative config blob from one or more veneer files",
+		Long:  `Generate a declarative config blob from one or more veneer files`,
```

Comment:
I'd consider adding a bit more documentation around this as, at least in general, it is a better user experience. Example:

https://github.com/operator-framework/operator-registry/blob/9e59597a435341d857283011c8ede1c16e5e292b/cmd/opm/alpha/bundle/build.go#L25